### ISSUE TITLE
Copying SparseMatrixLNK and ExtendableSparseMatrix is extremely slow.

### ISSUE THREAD

**Issue by moritzdrechselgrau** [WIAS-PDELib/ExtendableSparse.jl#14]
Copying a `SparseMatrixLNK` or `ExtendableSparseMatrix` object is orders of magnitudes slower than copying the corresponding `SparseMatrixCSC` object.
```julia
using SparseArrays, ExtendableSparse
Xcsc = sprand(10_000,10_000,0.01)
Xlnk = SparseMatrixLNK(Xcsc)
Xext = ExtendableSparseMatrix(Xcsc)
@time copy(Xcsc)
@time copy(Xlnk)
@time copy(Xext)
```
While copying `Xcsc` takes less than `0.01` seconds, copying `Xlnk` or `Xext` takes over `10` seconds. This is because no method for `Base.copy` is implemented. I will create a pull request that implements these methods similar to the method implemented for `SparseMatrixCSC` objects.

**Comment by j-fu** [comment]
Thanks for the PR - I am currently testing it, probably will tag a new version tonight.

**Comment by moritzdrechselgrau** [comment]
Great, thanks!

**Comment by j-fu** [comment]
Closed by #15
